packages/dynamodb-data-mapper/src/DataMapper.ts

```diff
@@ -306,6 +307,100 @@ export class DataMapper {
         }
     }
 
+    private delay(duration: number) {
+        return new Promise(resolve => setTimeout(resolve, duration));
+    }
+
+    /**
+     * Perform a UpdateTable operation using the schema accessible via the
+     * {DynamoDbSchema} property and the table name accessible via the
+     * {DynamoDbTable} property on the prototype of the constructor supplied.
+     *
+     * The promise returned by this method will not resolve until the table is
+     * active and ready for use.
+     *
+     * @param valueConstructor  The constructor used for values in the table.
+     * @param options           Options to configure the CreateTable operation
+     */
+    async createGlobalSecondaryIndex(
+        valueConstructor: ZeroArgumentsConstructor<any>,
+        {
```

Comment:
Could this parameter come last? Having an options bag precede a scalar argument seems odd.

Reply:
Great point - yes, that does seem more clear